core/src/codecs/default/DefaultVectorsFormat.cpp

```diff
@@ -157,6 +181,32 @@ DefaultVectorsFormat::read_uids(const storage::FSHandlerPtr& fs_ptr, std::vector
         const auto& path = it->path();
         if (path.extension().string() == user_id_extension_) {
             read_uids_internal(fs_ptr, path.string(), uids);
+            break;
+        }
+    }
+}
+
+void
+DefaultVectorsFormat::read_vectors(const storage::FSHandlerPtr& fs_ptr, knowhere::BinaryPtr& raw_vectors) {
+    const std::lock_guard<std::mutex> lock(mutex_);
+
+    std::string dir_path = fs_ptr->operation_ptr_->GetDirectory();
+    if (!boost::filesystem::is_directory(dir_path)) {
```

Comment:
don't use boost::filesystem directly here, since fs_ptr may be disk or s3 or hdfs, etc.
You need add an interface named "bool exist(std::string)" in storage/Operations.h
If this kind of change is too much, file a seperate issue to do this.

Reply:
It is better to create a new issue.

Reply:
This PR will not deal with that issue, maybe another issue will do.